presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixTableProperties.java

```diff
@@ -109,6 +110,11 @@ public PhoenixTableProperties()
                         TTL,
                         "Number of seconds for cell TTL.  HBase will automatically delete rows once the expiration time is reached.",
                         null,
+                        false),
+                stringProperty(
```

Comment:
Can we have a enum property for this like we define `STORAGE_FORMAT_PROPERTY` for HiveTableProperties

Reply:
In both cases I just followed the existing code. I think if we change that, it is for all other parts too, and then we're mixing concerns of PRs.

Reply:
Let me know if you want me to fix all the other instances as well in this PR or file a separate one.

Reply:
It would be better if we could fix all the other instances also. Maybe we can have s separate PR for them ?

Reply:
I fixed the Optional use below. The property list change should be a separate PR IMHO.